src/components/FixedGrid/GridBase.js

```diff
@@ -7,25 +7,33 @@ import Row from './Row/Row';
 import DroppableCell from './DroppableCell/DroppableCell';
 import DraggableItem from './DraggableItem/DraggableItem';
 import styles from './GridBase.module.css';
+import { useMemo } from 'react';
 
 function GridBase(props) {
   const {
     className,
     columns,
     dragAndDropEnabled,
-    items,
     onItemDrop,
-    order,
     renderEmptyCell,
     renderItem,
     rows,
     page,
+    gridState,
     ...other
   } = props;
 
+  const { items, order } = gridState;
+
   const gridClassName = classNames(styles.root, className);
 
-  const grid = utils.sortGrid({ columns, rows, order, items });
+  const grid = useMemo(
+    () => {
+      console.log('REmemoized');
+      return utils.sortGrid({ columns, rows, order, items });
+    },
+    [columns, rows, order, items]
+  );
```

Comment:
This `useMemo`  is not working as expected because the `gridState` is defined on the ´Board.component´ as an inline component. It gets recreated when the BoardComponent is rendered. The memozation of this `gridState` is not avaible beacuse ´Board´ is a class componente and the ´useMemo is not avaliable´.  @tomivm @shayc  what do you think?

Reply:
Do you think this change is necessary in this PR? I think that we can treat it separately and merge this PR without inconvenience.
Let me know!

Reply:
We can leave this to other PR where we improve the grid performance. I will create a new issue to remember to work on this

Reply:
Anyway, I tested it without `useMemo` and noticed more re-renders than when we use `useMemo`, so I’ll leave it.

Reply:
Fine Rodri!! 🚀